### PR TITLE
Add a few chnages

### PR DESCRIPTION
**Pull request by mdwalters**
- Better modal transition
- Fix typing indicator
- Some others

### CODE REVIEW

**Comment by mdwalters** [comment]
@CST1229

File: src/lib/Post.svelte [hunk 1]
```diff
@@ -38,17 +38,16 @@
...
 		"https://http.meower.org/",
 		"https://assets.meower.org/",
-		"https://api.meower.org/",
 		"https://forums.meower.org/",
 		// not everyone can add urls to go.meower.org, should be fine
 		"https://go.meower.org/",
 		"https://nextcloud.meower.org/",
 
-		// cubeupload
+		// Cubeupload
```
Comment (CST1229) [inline src/lib/Post.svelte:46]: cubeupload is the official name of the service
image

```suggestion
		// cubeupload
```

File: src/lib/Post.svelte [hunk 2]
```diff
@@ -237,14 +236,14 @@
...
 	<div class="post-images">
 		{#each images as { title, url }}
-			<a href={url} target="_blank"
-				><img
+			<a href={url} target="_blank">
+                <img
 					src={url}
 					alt={title}
-					title="{title} ({url})"
+					title={title}
```
Comment (CST1229) [inline src/lib/Post.svelte:243]: i was gonna say something about this but i realized you can see the image url in the post

**Comment by CST1229** [comment]
also i'll need to see how the modal transition looks

**Comment by CST1229** [comment]
how about this?

https://user-images.githubusercontent.com/68464103/214486634-7d564ecb-5475-40ee-a59c-27f46178f69b.mp4

(for comparison, here are the current and this pr's transitions)

https://user-images.githubusercontent.com/68464103/214486720-cbc06208-7522-4a1e-85d3-84a7a1f1c1b2.mp4

https://user-images.githubusercontent.com/68464103/214486895-9da6040d-8ba5-4cfe-a5a3-b75db3b53f57.mp4

**Comment by mdwalters** [comment]
> how about this?
> 
>  Jt1Xmx9Ooj.mp4 
> (for comparison, here are the current and this pr's transitions)
> 
>  k6rhFJaNf6.mp4 
>  7j7Da9t7Nj.mp4

YES

**Comment by mdwalters** [comment]
@CST1229 what should be the duratation of the scale thing?

**Comment by CST1229** [comment]
> @CST1229 what should be the duratation of the scale thing?

i think it was like 200ms for the modal scale, and 100ms for the background fade

**Comment by mdwalters** [comment]
ok

**Comment by mdwalters** [comment]
i feel like the end transition duratation needs to be prob the same as the starting duratation

**Comment by CST1229** [comment]
> i feel like the end transition duratation needs to be prob the same as the starting duratation

it should already be the same

**Comment by mdwalters** [comment]
idk anymore
[Screen recording 2023-01-25 10.20.27 AM.webm](https://user-images.githubusercontent.com/67456566/214602448-be9cb8c6-8b35-4c78-96a6-572324763331.webm)

**Review by silvxrcat — APPROVED** [review]
epic

**Pull request merged**